util/client/configuration.go

```diff
@@ -12,6 +12,9 @@ var QueryTimeoutSeconds int64
 // GenomicAnnotationsQueryTimeoutSeconds is the timeout for the client query in seconds (default to 10 seconds)
 var GenomicAnnotationsQueryTimeoutSeconds int64
 
+// GetStatusTimeoutSeconds is the timeout for the client query in seconds (default to 60 seconds)
```

Comment:
"for the client query" -> forgot to update the comment

Reply:
yep